src/lightning_app/runners/cloud.py

```diff
@@ -107,10 +117,41 @@ def dispatch(
                     preemptible=work.cloud_compute.preemptible,
                     shm_size=work.cloud_compute.shm_size,
                 )
+
+                drive_specs: List[V1LightningworkDrives] = []
+                for drive in work.drives:
+                    if drive.protocol == "lit://":
+                        drive_type = V1DriveType.NO_MOUNT_S3
+                        source_type = V1SourceType.S3
+                    elif drive.protocol == "s3://":
+                        drive_type = V1DriveType.INDEXED_S3
+                        source_type = V1SourceType.S3
+                    else:
+                        drive_type = V1DriveType.UNSPECIFIED
```

Comment:
This shouldn't happen. Every drive has a protocol.

Reply:
fixed. This now raises an error.